src/Exception/CacheException.php

```diff
@@ -13,7 +13,7 @@ public function __construct(
         private string $key,
         string $message = '',
         int $code = 0,
-        Throwable $previous = null
+        Throwable|null $previous = null
```

Comment:
Maybe `?Throwable $previous = null`?

Reply:
I kept it with the same style of the package code, which uses type union, which is explicit.